api/v1alpha1/airflowbase_types.go

```diff
@@ -293,6 +293,8 @@ type AirflowUISpec struct {
 	// Resources is the resource requests and limits for the pods.
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	EnableRoutes string `json:"enableroutes,omitempty"`
```

Comment:
Can you change this to `bool`?

Also, make sure you set it to default to `false` similarly to `ForceRoot`

Reply:
Changed